### ISSUE TITLE
Command ending in \ doesn't continue correctly (readline shell)

### ISSUE THREAD

**Issue by asmeurer** [xonsh/xonsh#693]
Reported this IRL to @scopatz today. Example:

``` xonsh
🐚 echo test\
test\
🐚
```

**Comment by laerus** [comment]
the behavior atm is different 
```
>>> echo test\
... secondline
... 
test\
xonsh: subprocess mode: command not found: secondline
```

**Comment by scopatz** [comment]
Potential fix in #2194

**Comment by scopatz** [comment]
Please feel free to reopen, if needed.
